src/bin/hive_chain.rs

```diff
@@ -71,6 +79,9 @@ async fn main() -> eyre::Result<()> {
 
     // Set the chain id
     let chain_id = starknet_provider.chain_id().await?;
+    let chain_id_mod = apply_chain_id_modulo(chain_id);
+
+    // Initialize the chain ID globally
     let modulo = (1u64 << 53) - 1;
```

Comment:
Remove the lines here since you already have the chain id mod

Reply:
Okay